### DISCUSSION TITLE
Can enterprise application use "Free" price plan?

### DISCUSSION THREAD

**Discussion by namdien177** [cypress-io/cypress#16413] [Questions and Help] [answered]
Can I use Cypress with Free Price Plan for enterprise applications (government application-related)?

I know that Cypress is under MIT license but it also provides an "Enterprise" plan so I just want to ask if it's a "must" to use "Enterprise" instead of a "Free" price plan.

**Comment by lmiller1990** [comment, marked as answer]
It's an MIT project, you can do whatever you like with it. The pricing page and enterprise plan is for our [dashboard product](https://www.cypress.io/dashboard/). Very useful for large orgs with huge test suites.
